src/main/java/seedu/address/model/job/JobDescription.java

```diff
@@ -1,20 +1,42 @@
 package seedu.address.model.job;
 
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
 /**
  * Represents a Job's description in the address book.
  */
 public class JobDescription {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Job description should not be blank";
+
+    /**
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "^\\S.*";
```

Comment:
I am not very good with regex, just to confirm, does this regex allow " " as input description?

Reply:
Hahahah I'm also not very good at regex, I just chatgpt and kept re-prompting/trying until one that works appeared. 

Whitespaces as input shown below. 

<img width="426" alt="image" src="https://github.com/user-attachments/assets/cc1b5879-d7c8-40f7-b945-3c5060987916">

Reply:
I have changed it to not use regex. Simpler to understand anyways. The contains '/' check is because I can do something like d/r/reqs and then description becomes "r/reqs".

<img width="472" alt="image" src="https://github.com/user-attachments/assets/99621ee6-f3dc-45af-90be-485b61329acb">

Reply:
The new test does not stop " " from being valid, which is not the expected behavior stated in the JavaDoc.